crates/node/src/lib.rs

```diff
@@ -446,24 +446,28 @@ impl Node {
             };
             let actions = vec![action];
 
-            drop(
-                self.ctx_manager
-                    .propose(
-                        context_id,
-                        executor_public_key,
-                        proposal_id.clone(),
-                        actions,
-                    )
-                    .await,
-            );
+            self.ctx_manager
+                .propose(
+                    context_id,
+                    executor_public_key,
+                    proposal_id.clone(),
+                    actions.clone(),
+                )
+                .await
+                .map_err(|e| {
+                    error!(%e, "Failed to create proposal {:?} with actions {:?} in context {:?} by identity {:?}", proposal_id, actions, context_id, executor_public_key);
```

Comment:
just failed to create proposal, I think is fine (to avoid spamming the console).. the error (from the contract) will be in `e`